### ISSUE TITLE
Error "could not create converter for windows-1252"

### ISSUE THREAD

**Issue by flemmens** [kosmtik/kosmtik#90]
I get this error when I try to open a standard osm carto css project :
"could not create converter for windows-1252"

The project runs fine with Tilemill on the same machine. The carto css used is this one:
https://github.com/gravitystorm/openstreetmap-carto

**Comment by yohanboniface** [comment]
Humm, I seem to be able to load the project without issue. Maybe it depends on some zoom/place.

Sounds like a shapefile error. Can you try narrowing which layer is causing the issue?

**Comment by flemmens** [comment]
Actually the error is displayed only at the launch zoom level (in this case zoom 4), all others zoom levels are correctly displayed, but I am unable to display the zoom 4 (the map remains empty), even if I reload the data.

**Comment by pnorman** [comment]
It will be the natural earth place shapefile, which is only used on low zooms.

**Comment by yohanboniface** [comment]
Indeed. If I remove the "encoding" key from this layer, I don't have any error any more.

Encoding seems to be here from [ages](https://github.com/gravitystorm/openstreetmap-carto/commit/c87ca4e4) so I wonder why the error only raises now. And I don't know why it works for me without the encoding set to `window-1252` (I think Mapnik defaults to `utf8`).

Also, I'm not sure if the error message means `not able to open the shapefile with encoding xxx`, or, more likely, `shapefile plugin is not able to work with windows-1252 on this machine`.

The file really seems to be full ascii: I see not accent. So maybe forcing encoding in there is useless anyway?

@pnorman thoughts?

**Comment by yohanboniface** [comment]
See https://github.com/mapnik/mapnik/issues/1476

**Comment by yohanboniface** [comment]
Also issue in Tilemill https://github.com/mapbox/tilemill/issues/2525

**Comment by yohanboniface** [comment]
@springmeyer given that the issue seems to araise recently while the encoding is set from a long time (https://github.com/gravitystorm/openstreetmap-carto/commit/c87ca4e4), could this be related to recent Mapnik 3.0.0 builds that people started to use? Bringing this to your attention just in case in ring any bell to you.

**Comment by pnorman** [comment]
That file is not purely ASCII.

As distributed, the Natural Earth shapefile has an encoding error. To fix this, the osm-carto download script runs ogr2ogr on it, fixing it up.

One possible cause is that this step was missed, but it shouldn't be required with Mapnik >= 2.2.

Another is that for some reason, the Mapnik that your Kosmtik install is using doesn't support 1252.

**Comment by flemmens** [comment]
I tried with the ogr fix but the issue is still there, and as you said normally it is not necessary with mapnik > 2 (the version used by my kosmtik install is 3.0.0). However I also have the latest version (3.0.4) installed, maybe is there a conflict somewhere ? Anyway, this is not a big deal, I can use alternative sources than Natural Earth data.

**Comment by ROlda** [comment]
not convert, but for view:

<Parameter name="encoding">cp852</Parameter>
